services/landingpage/compose.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+  landingpage:
+    image: ghcr.io/scicatproject/landingpageserver:latest
```

Comment:
the tag will need to be reviewed after https://github.com/SciCatProject/scicatlive/issues/183 is closed. Fine to keep it as is for now

also `config` might be added, together with the folder and the related files